shared_infra/Makefile

```diff
@@ -0,0 +1,17 @@
+ROOT = $(shell git rev-parse --show-toplevel)
+
+ifneq ($(ROOT), $(shell pwd))
+include $(ROOT)/shared.Makefile
+endif
+
+
+## Run a plan on main stack
+terraform-shared-plan: uptodate-git .docker/terraform_ci
```

Comment:
Because `uptodate-git` is still defined in the top-level Makefile, I don't think this will work if you're running inside the `shared_infra` directory. Is that intentional?

Ditto `.docker/terraform_ci`.